tools/eval_net.py

```diff
@@ -218,7 +218,7 @@ def eval(cfg, args, model, weights_basename, criterion, best_value):
         and angle is None
         and translation is None
         and nshots is None
-        and trajectory == "None"
+        and trajectory == "none"
```

Comment:
why are all of these fields gated by an if statemnt? are you trying to set default values?

Reply:
Yes, just setting default values here.